content/posts/2024-11-05-flang-new.md

```diff
@@ -0,0 +1,444 @@
+---
+author: "David Spickett"
+date: "2024-11-05"
+tags: ["Flang", "MLIR"]
+title: "LLVM Fortran Levels Up: Goodbye flang-new, hello flang!"
+---
+
+LLVM has included a Fortran compiler "Flang" since LLVM 11. What you might not
+know is that the name of Flang's binary was not `flang`, but `flang-new`.
```

Comment:
Note "LLVM Flang" includes multiple binaries, `flang` (fka `flang-new`) happens to be Flang's compiler driver. However, when "LLVM Flang" was merged into LLVM, `flang` was not a binary ;-)

Reply:
I will find a way to reword this such that it does not imply that *in llvm 11* there was a flang-new, but that just prior to llvm 20 you would have found only flang-new.

Reply:
Done.